ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -41,6 +41,22 @@ def mask2label(path_label, aim=0):
     return list_label_image
 
 
+
+
+
+
```

Comment:
why these empty lines?

Reply:
Thanks for the comment, they are removed.